src/fuse.js

```diff
@@ -336,6 +336,19 @@
     keys: []
   };
 
+  Fuse.prototype.add = function(to_add) {
+    if (!Utils.isArray(to_add)) {
+      to_add = [to_add];
+    }
+    this.list = this.list.concat(to_add);
+  };
+
+  Fuse.prototype.remove = function(to_remove) {
```

Comment:
Shouldn't this remove by index instead? If it's by string, it won't cover all the cases.